copilot/data-store-celery/addons/data-store-find-data-access.yml

```diff
@@ -0,0 +1,45 @@
+Parameters:
+  App:
+    Type: String
+    Description: Your application's name.
+  Env:
+    Type: String
+    Description: The environment name your service, job, or workflow is being deployed to.
+  Name:
+    Type: String
+    Description: The name of the service, job, or workflow being deployed.
+Resources:
+  datastorefinddataAccessPolicyForCelery:
+    Type: AWS::IAM::ManagedPolicy
+    Properties:
+      Description: Grants read/write access to data store's find-data S3 bucket.
+      PolicyDocument:
+        Version: '2012-10-17'
+        Statement:
+          - Sid: S3ObjectActions
+            Effect: Allow
+            Action:
+              - s3:GetObject
+              - s3:PutObject
+              - s3:PutObjectACL
+              - s3:PutObjectTagging
+              - s3:DeleteObject
+              - s3:RestoreObject
```

Comment:
Can we minimise this permissions set? i.e. I don't think the service should need to delete or restore?